README.md

```diff
@@ -2,3 +2,12 @@
 A synchronization daemon for [AsteroidOS](http://asteroidos.org/) watches. Forked from [Rockpool](https://github.com/abranson/rockpool).
 
 asteroidsyncserviced is a daemon relying on libasteroid to synchronize data from a given platform to a watch. It comes with a QML module named asteroidsyncservice. It is used by starfish, a SailfishOS synchronization app and Telescope, an Ubuntu Touch synchronization app.
+
+## Building
+
+To build asteroidsyncservice, all submodules must be loaded.  
+   `git submodule update --init`
```

Comment:
Please rebase this on the current master branch (which already includes a commit modifying the README with this) for your PR to be mergeable

Reply:
Done.